public/app/plugins/datasource/mysql/mySqlMetaQuery.ts

```diff
@@ -0,0 +1,60 @@
+import MySQLQueryModel from './MySqlQueryModel';
```

Comment:
nit: mySqlMetaQuery.ts -> MySqlMetaQuery.ts

Reply:
it's lower case because its not a class/exporting a class